tests/components/BeneficiairesMultiselectAutocomplete.test.tsx

```diff
@@ -0,0 +1,240 @@
+import { render, screen, within } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+import BeneficiairesMultiselectAutocomplete, {
+  OptionBeneficiaire,
+} from 'components/jeune/BeneficiairesMultiselectAutocomplete'
+import { ListeDeDiffusion } from 'interfaces/liste-de-diffusion'
+
+describe('BeneficiairesMultiselectAutocomplete', () => {
+  let beneficiaires: OptionBeneficiaire[]
+  let listes: ListeDeDiffusion[]
+  let onUpdate: (selectedIds: {
+    beneficiaires?: string[]
+    listesDeDiffusion?: string[]
+  }) => void
+
+  let input: HTMLElement
+  let options: HTMLElement
+  beforeEach(async () => {
+    // Given
+    beneficiaires = [
+      { id: 'option-1', value: 'Option 1' },
+      { id: 'option-2', value: 'Option 2' },
+      { id: 'option-3', value: 'Option 3' },
+    ]
+    listes = [
+      {
+        id: 'liste-1',
+        titre: 'Liste 1',
+        beneficiaires: [
+          {
+            id: 'option-1',
+            nom: 'Option',
+            prenom: '1',
+            estDansLePortefeuille: true,
+          },
+        ],
+      },
+      {
+        id: 'liste-2',
+        titre: 'Liste 2',
+        beneficiaires: [
+          {
+            id: 'option-2',
+            nom: 'Option',
+            prenom: '2',
+            estDansLePortefeuille: true,
+          },
+          {
+            id: 'option-3',
+            nom: 'Option',
+            prenom: '3',
+            estDansLePortefeuille: true,
+          },
+        ],
+      },
+    ]
+    onUpdate = jest.fn()
+
+    // When
+    render(
+      <BeneficiairesMultiselectAutocomplete
+        beneficiaires={beneficiaires}
+        listesDeDiffusion={listes}
+        typeSelection='Bénéficiaires'
+        onUpdate={onUpdate}
+      />
+    )
+
+    // Then
+    input = screen.getByRole('combobox', {
+      name: 'Rechercher et ajouter des bénéficiaires Nom et prénom',
```

Comment:
```suggestion
      name: 'Rechercher et ajouter des destinataires Nom et prénom',
```
+ cf ligne 216, mettre à jour le helpText: "Nom et prénom du bénéficiaire ou nom de votre liste de diffusion"
cf. maquette/notion